install_custom_build_configuration/after/SampleApp.xcodeproj/project.xcworkspace/xcshareddata/SampleApp.xccheckout

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
does CocoaPods generate this or was it from opening the project in Xcode? if the latter, can we delete it?

Reply:
my bad -- cocoapods does _not_ generate it. will ignore xccheckout files